docker/texlive-profile.txt

```diff
@@ -0,0 +1,10 @@
+selected_scheme scheme-minimal
+TEXDIR /usr/local/texlive
+TEXMFCONFIG ~/.texlive/texmf-config
+TEXMFHOME ~/texmf
+TEXMFLOCAL /usr/local/texlive/texmf-local
+TEXMFSYSCONFIG /usr/local/texlive/texmf-config
+TEXMFSYSVAR /usr/local/texlive/texmf-var
+TEXMFVAR ~/.texlive/texmf-var
```

Comment:
I don't you need to configure all these? If want to why and why default one doesn't works?

Reply:
To know precisely where everything is located in the container. It is more or less a default profile file.

Reply:
I think it would be good to remove them though.

Reply:
No, I would like to have the paths configured here. I want to have a place where I can easily find out how the tex installation is configured.